data/notices.json

```diff
@@ -471,6 +471,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(events-targets): Hardcoded AWS Partition in ECS task resource ARN",
+      "issueNumber": 29610,
+      "overview": "Partition in policy statement for `ecs:TagResource` action in `EcsTask` was hardcoded to `aws` instead of being resolved dynamically in v2.132.0, resulting in failure to deploy in any partition escept AWS with an IAM validation failure on the partition arn.",
```

Comment:
```suggestion
      "overview": "Partition in policy statement for `ecs:TagResource` action in `EcsTask` was hardcoded to `aws` instead of being resolved dynamically in v2.132.0, resulting in failure to deploy in any partition except AWS with an IAM validation failure on the partition arn.",
```